### PR TITLE
Clarify role and usage of subnet and FIP config opts

### PR DESCRIPTION
**Pull request by johnsca**
For the OpenStack integration documentation, the role and distinction between the four config options related to load balancer FIP network and tenant subnet info isn't clearly spelled out and was leading to confusion.

### CODE REVIEW

**Comment by johnsca** [comment]
[Direct link to preview of changed page](https://deploy-preview-618--cdk-docs-next.netlify.app/kubernetes/docs/openstack-integration)

**Comment by johnsca** [comment]
@ajkavanagh Would you mind taking a look over this as well and make sure it seems accurate?

File: pages/k8s/openstack-integration.md [hunk 1]
```diff
@@ -51,6 +51,111 @@ relations:
...
+are defined with `type=LoadBalancer`, and as a replacement for the load balancer in front of the
+API server itself.
+
+In either case, the load balancers can optionally have floating IPs (FIPs) attached to them to
+allow for external access.
+
+<div class="p-notification--caution">
+  <p markdown="1" class="p-notification__response">
+    <span class="p-notification__status">Note:</span>
+For security reasons, the SGs automatically managed by Juju will not by default allow traffic into
```
Comment (ajkavanagh) [inline pages/k8s/openstack-integration.md:92]: Probably need to spell out "SG" as Security Group for its first use?

File: pages/k8s/openstack-integration.md [hunk 2]
```diff
@@ -51,6 +51,111 @@ relations:
...
+```
+```
+Hello Kubernetes!
+```
+
+
+#### API Server Load Balancer
+
 If desired, the openstack-integrator can also replace kubeapi-load-balancer and create a native OpenStack
 load balancer for the Kubernetes API server, which both reduces the number of machines required and is
```
Comment (ajkavanagh) [inline pages/k8s/openstack-integration.md:160]: I know it's not part of the review, but the 'both' in the above service looks superfluous?

**Review by ajkavanagh — APPROVED** [review]
LGTM

**Review by evilnick — APPROVED** [review]
:cake:

**Pull request merged**